dspace-api/src/main/java/org/dspace/app/mediafilter/MediaFilterServiceImpl.java

```diff
@@ -366,6 +377,11 @@ public boolean processBitstream(Context context, Item item, Bitstream source, Fo
             }
         } catch (OutOfMemoryError oome) {
             System.out.println("!!! OutOfMemoryError !!!");
+            try {
+                if (destStream != null) {
+                    destStream.close();
+                }
+            } catch (IOException ex) {}
```

Comment:
The above try-block can be replaced by org.apache.commons.io.IOUtils#closeQuietly(java.io.InputStream)